src/QuokkaSimulation.hpp

```diff
@@ -1721,7 +1721,7 @@ void QuokkaSimulation<problem_t>::subcycleRadiationAtLevel(int lev, amrex::Real
 			PrintRadEnergySource(radEnergySource);
 
 			// Deposit radiation from all particles that have luminosity. When there are no particles with luminosity, this will do nothing.
-			AMRSimulation<problem_t>::particleRegister_->depositRadiation(radEnergySource, lev, time_subcycle);
+			AMRSimulation<problem_t>::particleRegister_.depositRadiation(radEnergySource, lev, time_subcycle);
```

Comment:
If you add `using AMRSimulation<problem_t>::particleRegister_;` at the start of the `QuokkaSimulation` class, you will be able to reference `particleRegister_` as a normal class variable here.

Reply:
Done.